src/hooks/v2v3/transactor/MintTokensTx.ts

```diff
@@ -55,7 +56,7 @@ export function useMintTokensTx(): TransactorInstance<{
     } catch {
       const missingParam = !transactor
         ? 'transactor'
-        : !JBController
+        : JBController
```

Comment:
**question** whats the reason for this change?

Reply:
thank youuu! dumb mistake